packages/http-client-csharp/emitter/src/lib/logger.ts

```diff
@@ -14,9 +14,9 @@ export class Logger {
   private level: LoggerLevel;
   private program: Program;
 
-  public constructor(program: Program, level: LoggerLevel) {
+  public constructor(program: Program, level?: LoggerLevel) {
```

Comment:
why make level optional?

Reply:
well I think it could simplify it - we are taking Info as default value, if we do it inside the ctor, it could be simpler

Reply:
well maybe it should be better in the previous way.
I will change this back